client/src/app/app.module.ts

```diff
@@ -78,7 +82,7 @@ type StoreType = {
     HttpModule,
     HttpClientModule,
     RouterModule.forRoot(ROUTES, {
-      useHash: Boolean(history.pushState) === false,
+      useHash: Boolean(history.pushState) === true,
```

Comment:
please change to `useHash: true`, as your server is not set to serve the same app by different routes